data-jdbc/src/test/java/io/micronaut/data/jdbc/oraclexe/OracleXEJsonEntityRepository.java

```diff
@@ -1,9 +1,21 @@
 package io.micronaut.data.jdbc.oraclexe;
 
+import io.micronaut.data.annotation.Query;
 import io.micronaut.data.jdbc.annotation.JdbcRepository;
 import io.micronaut.data.model.query.builder.sql.Dialect;
 import io.micronaut.data.tck.repositories.JsonEntityRepository;
 
+import java.util.Optional;
+
 @JdbcRepository(dialect = Dialect.ORACLE)
 public interface OracleXEJsonEntityRepository extends JsonEntityRepository {
+
+    @Query("SELECT JSON_VALUE(JSON_BLOB, '$.description') FROM JSON_ENTITY WHERE id = :id")
```

Comment:
This would not work for BLOB field (returns NULL) without fix provided in the PR.